src/event/handlers/CodeblocksOverFileUploadsHandler.ts

```diff
@@ -9,10 +9,14 @@ class CodeblocksOverFileUploadsHandler extends EventHandler {
 
 	async handle(message: Message): Promise<void> {
 		let invalidFileFlag = false;
+		let invalidFileExtension: string = "";
 
 		if (message.attachments.size > 0) {
 			message.attachments.forEach(attachment => {
-				if (!ALLOWED_FILE_EXTENSIONS.includes(attachment.name?.split(".").pop()!)) {
+				const fileExtension = attachment.name?.split(".").pop()!.toLowerCase();
+
+				if (!ALLOWED_FILE_EXTENSIONS.includes(fileExtension)) {
+					invalidFileExtension = fileExtension;

```

Comment:
This assumes that only one file attachment can be sent per message, although we have a map of attachments... Anyway, I didn't find a way of sending a message with more than one file attachment yet.